docassemble/ssioverpaymentwaiver/data/questions/ssi_overpayment_waiver.yml

```diff
@@ -1323,6 +1335,17 @@ code: |
 code: |
   expense_support_details = comma_and_list([item.support_explanation for item in expenses if item.type == "support"])
 ---
+id: preview pdf
+continue button field: preview_pdf
+question: |
+  Review your form before you sign it
+subquestion: |
+  Here is a preview of the form you'll sign on the next page. 
+  
+  ${ ssa_634_template if only_change_monthly_due else ssa_632_template }
+   
+  Remember to come back to this window to continue and sign your form.
```

Comment:
Adding a preview here with `skip undefined: True` means that the PDF will never have the signature block added.

We need to add a `reconsider` or move to ALDocument or a DADict with multiple "versions" of the output.

Reply:
Good catch, didn't realize that. I'll add a reconsider, I didn't want to try to refactor to use ALDocument when it's not depending on any AL repo yet.

Reply:
The DADict is a pretty lightweight patch if you want to go that route.